bdd-for-c.h

```diff
@@ -41,6 +41,23 @@ SOFTWARE.
 #include <Windows.h>
 #include <io.h>
 #define __BDD_IS_ATTY__() _isatty(_fileno(stdout))
+size_t strlcat(char* dst, const char* src, size_t size)
```

Comment:
It looks like this is the only dependency we drag from BSD headers on *nix systems so probably worth it to rename this to `__bdd_strlcat__`, compile unconditionally and use it everywhere. Could you please make this change?